editor/project_manager.cpp

```diff
@@ -474,7 +506,7 @@ class ProjectDialog : public ConfirmationDialog {
 					initial_settings["application/config/icon"] = "res://icon.png";
 					initial_settings["rendering/environment/default_environment"] = "res://default_env.tres";
 
-					if (ProjectSettings::get_singleton()->save_custom(dir.plus_file("/project.godot"), initial_settings, Vector<String>(), false)) {
+					if (ProjectSettings::get_singleton()->save_custom(dir.plus_file("/project.godot"), initial_settings, Vector<String>(), false) != OK) {
```

Comment:
I know it was already like this before, but `String::plus_file()` already adds a `/` if necessary, so it should not be added manually.